packages/pn-pa-webapp/src/utils/events.ts

```diff
@@ -1,3 +1,5 @@
+import { EventsType } from '@pagopa-pn/pn-commons/src/types/MixpanelEvents';
```

Comment:
Dovrebbe bastare @pagopa-pn/pn-commons

Reply:
corretto l'export in pn-commons e i vari import. nelle applicazioni